data/remnant missions.txt

```diff
@@ -495,3 +560,713 @@ mission "Remnant: Technology Available"
+			`	After taking a moment to enjoy the view, you shut the door as gently as you can, purge the ship with fresh air, and begin the journey back to Aventine.`
+	on complete
+		"reputation:  Drak " += 1
+		conversation
+			`Back on Aventine, you share the recording of the void sprite's behavior with Plume. "These Sprites demonstrate considerably more communication than we originally thought. And obviously they must have communicated what you did on Nasqueron. Interesting..." Plume's excited staccato exclamations trail off as he begins to make notes on a datapad he has on hand.`
+
+
+
+mission "Remnant: Learn Sign 1"
+	name "Learn Remnant Sign"
+	description `Visit each of the Remnant worlds to find ways to learn their sign language.`
+	minor
+	source "Caelian"
+	stopover "Aventine"
+	destination "Viminal"
+	to offer
+		has "Remnant: Technology Available: offered"
+		random < 50
+	on offer
+		conversation
+			`As you make your way through the spaceport, you become increasingly aware of how quiet it is. Looking around, you note that it doesn't seem any less busy than usual, just that only a handful of people are paying attention to you. As a result, they have reverted to their normal gestures instead of singing. It feels oddly isolating, and eerie, to realize that there are hundreds of conversations going on around you, yet you can't understand a single word of it.`
+			choice
+				`	(Ask someone to teach you.)`
+				`	(Think about asking later.)`
+					defer
+				`	(Ignore it.)`
+					decline
+					
+			`	Looking around the spaceport, you eventually find someone at what looks remarkably like an information desk.`
+			choice
+				`	"I would like to learn how to talk using signs."`
+					goto words
+				`	(Sing a short verse about not understanding people because your hands are still.)`
+					goto song
+			label words
+			`	She looks at you with a confused expression, her hands making a few quick gestures. After a minute she points at the hand scanner embedded in the desk.`
+				goto scan
+			label song
+			`	She looks surprised, but then recovers and sings a verse about how hands can learn but must be cautious, and gestures at the hand scanner embedded in the desk.`
+			label scan
+			`	As you place your hand on the scanner you feel a quick tingle as it scans your hand, then a pinprick as it extracts a drop of blood. The lady behind the desk looks at her screen. She nods, taps on the screen a few times, then retrieves a datapad from beneath the desk.`
+			`	She hands the pad to you, and sings a brief verse about perseverance and hope.`
+			`	Back on your ship you flip the pad on. The datapad appears to be loaded with a number of historic musicals and operas where the meaning of the lyrics has been replicated with gestures as subtitles. It is interspersed with what appear to be Remnant children's videos slowly explaining gestures and pausing to give you the opportunity to practice.`
```

Comment:
Remnant children will grow up learning signs so a children's video would be more likely to be teaching spoken word. Perhaps these videos could be explaining complex gestures.

Reply:
I was actually thinking of a culture in a book I read a while back that used some alternative communication stuff along the lines of sign, and the protagonist was being taught the language. One of the comments from his instructors was that communicating emotions publicly via facial expressions and laughter and stuff like that was something kids do; Talking and smiling is natural, and thus something intimate to be kept private within the family. 

I was thinking of developing the Remnant along the same lines. It has already been established that they consider their voices and singing to be mainly something done at home or for entertainment, it isn't a large stretch that children might start out learning that and then shift to sign as they grow older and deal with people outside their family more. Much like language classes teach vocabulary today, the Remnant education system would almost certainly have vocabulary videos.